README.md

```diff
@@ -8,19 +8,40 @@ Cumulus is a cloud-based data ingest, archive, distribution and management
 prototype for NASA's future Earth science data streams.
 
 Read the [Cumulus Documentation](https://cumulus-nasa.github.io/)
- 
+
+## Installation
+
+```bash
+nvm use
+npm install
+```
+
 ## Running tests locally
 
 These tests run against AWS, so a Cumulus deployment is needed. Set up the deployment using the configurations in this repository. Deployment instructions are located [here](https://cumulus-nasa.github.io/docs/deployment.html). The dashboard is not needed for these tests.
 
+### Using a different stack than `test-cumulus`
```

Comment:
I would remove this header since people should be discouraged from using the `test-cumulus` stack. The instructions should say configure your own stack.